README.md

```diff
@@ -1,37 +1,113 @@
+|  Number of people with minister role |     535    |
+
+\* Digital original parliamentary records for some years in the 1990s are not paginated and thus do not contribute to the page count.See also §_Number of Pages in Parliamentary Records_.
+
+### Parliamentary Records over time
+
+#### Number of Parliamentary Records
+
+![Number of Parliamentary Records](scripts/stats-dashboard/figures/n-prot/n-prot.png)
+
+#### Number of Pages in Parliamentary Records
+
+![Number of Pages in Parliamentary Records](scripts/stats-dashboard/figures/prot-pages/prot-pages.png)
+
+#### Number of Speeches in Parliamentary Records
+
+![Number of Speeches in Parliamentary Records](scripts/stats-dashboard/figures/prot-speeches/prot-speeches.png)
+
+Note: We are aware of an issue whereby speeches are over counted in the data's current form in the years after 2014 -- we're working on a fix. Until then, the following static graph is a better representation of the actual speeches in the Parliamentary Records for those years.
+
+![Static Number of Speeches](scripts/stats-dashboard/figures/prot-speeches/tomas.png)
+
+#### Number of Words in Parliamentary Records
+
+![Number of Words in Parliamentary Records](scripts/stats-dashboard/figures/prot-words/prot-words.png)
+
+### Members of Parliament over time
+
+![Members of Parliament over time](scripts/stats-dashboard/figures/mp-coverage/mp-coverage.png)
+
+## Quality assessment
+
+### Speech-to-speaker mapping
+
+We check how many speakers in the parliamentary records our algorithms idenify in each release. 
 
 ![Estimate of mapping accuracy](https://raw.githubusercontent.com/welfare-state-analytics/riksdagen-corpus/main/input/accuracy/version_plot.png)
 
-While the contents of the corpus will change due to curation and expansion, we aim to keep the deliverable API, the corpus/ folder, as stable as possible. This means we avoid relocating files or folders, changing formats, changing columns in metadata files, or any other changes that might break downstream scripts. Conversely, files outside the corpus/ folder are internal to the project. End users may find utility in them but we make no effort to keep them consistent.
+### Correct number of MPs over time
+
+![Ratio of MP to seats over time](scripts/stats-dashboard/figures/mp-coverage/mp-coverage-ratio.png)
+
+
 
-The data in the corpus is delivered as TEI XML files to follow established practices. The metadata is delivered as CSV files, following a normal form database structure while allowing for a legible git history. A more detailed description of the data and metadata structure and formats can be found in the README files in the corpus/ folder.
 
 ## Participate in the curation process
 
 If you find any errors, it is possible to submit corrections to them. This is documented in the [project wiki](https://github.com/welfare-state-analytics/riksdagen-corpus/wiki/Submit-corrections).
+
+
+## Acknowledgement of support
+
+- Westac funding: 2018-0606
```

Comment:
I think you should add Vetenskapsrådet (2018-0606) and Riksbankens Jubileumsfond (IN22-0003)

Reply:
resolved 3b9be805f8